pandas-stubs/__init__.pyi

```diff
@@ -8,7 +8,7 @@ from .core.frame import DataFrame as DataFrame
 from .core.series import Series as Series
 from .core.indexes import Index as Index
 
-_AxisType = Literal["columns", "index"]
+_AxisType = Literal["columns", "index", 0, 1]
```

Comment:
If you change this here, you should probably change the definition of `_AxisType` everywhere else as well...

Reply:
I will change it in `frame.pyi` and then import it into `__init__.pyi` from `frame`.